SU-SWS/stanford_profile
# D8CORE-859 Allow videos in cards

# READY FOR REVIEW

# Summary
- Allow video media type in cards

# Need Review By (Date)
- 12/11

# Urgency
- medium

# Steps to Test
1. checkout this branch and the same branch name in [jumpstart_ui](https://github.com/SU-SWS/jumpstart_ui/pull/10)
1. `drush cim`
1. add a basic page with a card paragraph
1. insert a youtube video
1. validate it looks like you'd expect.

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
